app/src/main/java/com/swent/assos/model/data/CommitteeMember.kt

```diff
@@ -0,0 +1,8 @@
+package com.swent.assos.model.data
+
+data class CommitteeMember(
+    val id: String = "",
+    val memberId: String = "",
+    val position: String = "",
+    val rank: Int = 0,
+)
```

Comment:
Wouldn't it be useful to also have the assoId ?

Reply:
bah non